docs/user-journeys/users/application-developer/intermediate.md

```diff
@@ -10,68 +10,102 @@ track: "USERS › APPLICATION DEVELOPER › INTERMEDIATE"
+#### Workloads
 
-Curabitur bibendum tempor mi, vel lacinia nisi vulputate ac. Nulla dignissim consectetur nisl nec tincidunt. Etiam pharetra facilisis sapien, non gravida velit fermentum sed. Ut ac ultrices nunc, in vestibulum urna. Suspendisse accumsan euismod felis, sit amet rhoncus neque volutpat luctus. Aliquam tincidunt pellentesque mauris, sed tempus diam. Mauris in elit eget justo tempor suscipit.
+{% glossary_tooltip text="Deployments" term_id="deployment" %}, which you previously covered, are ideal for running, updating, and scaling stateless web servers (e.g. a Rails app). However, for other workloads, you may find other Kubernetes API resources more useful:
+* **{% glossary_tooltip text="StatefulSets" term_id="statefulset" %}** - Because each Pod in a StatefulSet has a sticky identity, StatefulSets can be used to run datastores like Cassandra or Elasticsearch.
+* **{% glossary_tooltip text="DaemonSets" term_id="daemonset" %}** - Because they run per node, you may find DaemonSets useful for logging and monitoring.
+* **{% glossary_tooltip text="Jobs" term_id="job" %}** - You can use these for one-off tasks like running a script or setting up a work queue.
+* **{% glossary_tooltip text="CronJobs" term_id="cronjob" %}** - These are similar to Jobs, but allow you to schedule their execution (similarly to *crontab*).
 
-Aenean suscipit arcu ac leo tincidunt tempus. Donec maximus tellus libero, ac ullamcorper magna lobortis ac. Integer mollis nisl vitae magna gravida, nec ornare ex consectetur. Sed mattis tincidunt nisi, at consequat tellus malesuada non. Integer vel semper nisi, ut fringilla velit. Nam felis ex, congue non dui vitae, sollicitudin convallis turpis. Phasellus porttitor maximus turpis, in varius nibh fermentum aliquam. Cras finibus lacus non diam porttitor porttitor. Nulla fringilla sagittis nibh nec condimentum. Duis egestas mauris nec dolor hendrerit ullamcorper sit amet in mi. Phasellus sollicitudin justo diam.
+For more info, you can see the [full set of Kubernetes resource types](/docs/reference/kubectl/overview/#resource-types){:target="_blank"} and reference the [accompanying API spec](/docs/api-reference/v1.9/){:target="_blank"}.
```

Comment:
I wouldn't call this the "accompanying API spec". Because we have an OpenAPI spec [here](https://github.com/kubernetes/kubernetes/blob/master/api/openapi-spec/swagger.json). For this link target, I would say the "API reference docs".